src/porting.cpp

```diff
@@ -511,6 +512,12 @@ void initializePaths()
 		errorstream << "Failed to get one or more system-wide path" << std::endl;
 
 #endif
+#ifdef STATIC_LOCALEDIR
+	if (STATIC_LOCALEDIR[0] != '\0')
+		path_locale = STATIC_LOCALEDIR;
+	else
```

Comment:
I suggest removing the "else" to fix the indentation on the following code line.